.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+
+python:
+  - "2.7"
+  - "3.5"
+
+virtualenv:
+  system_site_packages: true
+
+script:
+  - make quality
```

Comment:
I think `install` step is missing? we need to install requirements first before using them for checking quality